src/PrizePool.sol

```diff
@@ -337,48 +337,47 @@ contract PrizePool is TieredLiquidityDistributor {
     emit WithdrawReserve(_to, _amount);
   }
 
-  /// @notice Allows the Manager to complete the current prize period and starts the next one, updating the number of tiers, the winning random number, and the prize pool reserve
+  /// @notice Allows the Manager to close the current open draw and open the next one.
+  ///         Updates the number of tiers, the winning random number and the prize pool reserve.
   /// @param winningRandomNumber_ The winning random number for the current draw
-  /// @return The ID of the completed draw
-  function completeAndStartNextDraw(
-    uint256 winningRandomNumber_
-  ) external onlyDrawManager returns (uint16) {
+  /// @return The ID of the closed draw
+  function transitionDraws(uint256 winningRandomNumber_) external onlyDrawManager returns (uint16) {
```

Comment:
I'm not sure we should convey that this function also starts the next Draw, it's kinda implied.
I feel like `completeDraw` or `awardDraw` would be enough.

Reply:
Yeah I originally changed it to just `closeDraw` but I figured I should retain the message that was conveyed by the old name. I'll change it back to `closeDraw`.